index.html

```diff
@@ -29,14 +29,14 @@
 
   <header class="header" data-header>
     <div class="container header__container">
-      <a href="#" class="logo">
-        <img src="./assets/images/logo.webp" width="136" height="46" alt="Proktomed" style="background: white;" />
+      <a href="#">
+        <img class="header__container-logo" src="./assets/images/logo.webp" width="136" height="46" alt="Proktomed" />
```

Comment:
Using webp alone is not the best practice, as it may not work eg. on this year's Safari. So the best way is just a compressed png, or png/avif combo